### DISCUSSION TITLE
Working with voice file

### DISCUSSION THREAD

**Discussion by mrddter** [telegraf/telegraf#1950] [Q&A]
Hi all,

I know this isn't directly related to telegraf, but I figure you might have encountered a similar need.
I have to capture a voice file as a WAV stream instead of an OGG/OGA file. 

Do you know a convenient way to do this on the fly? Thanks in advance.

**Comment by MKRhere** [comment]
Use a package like https://www.npmjs.com/package/@mmomtchev/ffmpeg to stream the conversion.

**Reply by mrddter** [reply]
thanks
